src/main/kotlin/mb/coronium/plugin/FeaturePlugin.kt

```diff
@@ -136,13 +126,13 @@ class FeaturePlugin : Plugin<Project> {
     val featureXmlDir = project.buildDir.resolve("featureXml")
     val featureXmlTask = project.tasks.create("featureXmlTask") {
       // Depend on (files from) bundle configuration because dependencies in this configuration affect the feature model.
-      dependsOn(bundleConfig)
       inputs.files(bundleConfig)
       val featureXmlFile = featureXmlDir.resolve("feature.xml").toPath()
       outputs.file(featureXmlFile)
       doLast {
+        val mergedFeature = feature.mergeWith(bundleConfig)
```

Comment:
Instead of adding the buildConfig.dependencies to the feature, Gradle waits until task execution and _then_ merges the feature (which contains the parsed feature.xml) with the dependencies in the bundleConfig. Finally, the _merged_ feature is written to feature.xml.